### ISSUE TITLE
Show total number of steps after current step count

### ISSUE THREAD

**Issue by aiday-mar** [microsoft/vscode#229562]
Testing #229460

I notice that as we cycle through the accessible view, the text mentions the number of the current step. I am wondering if it would be helpful for screen reader users to write also the total number of steps? For example this way:

`Step 2 out of 5`

This makes it clearer there are more steps left to read, and prompts them to verify if there are buttons that could help them cycle further.

**Comment by bhavyaus** [comment]
Removed all mention of steps since we don't want to call this out as an ordered list which should be navigated in a specific order. This also allows extensions to create their own step logic. See https://github.com/microsoft/vscode/issues/230334 for an example.
